app/models/manageiq/providers/autosde/storage_manager/autosde_client.rb

```diff
@@ -1,8 +1,8 @@
-require 'autosde_oas_client/generated/lib/autosde_openapi_client'
+require 'autosde_openapi_client'
 
-class ManageIQ::Providers::Autosde::StorageManager::AutosdeClient < OpenapiClient::ApiClient
+class ManageIQ::Providers::Autosde::StorageManager::AutosdeClient < AutosdeOpenapiClient::ApiClient
   include Vmdb::Logging
-  include OpenapiClient
+  include AutosdeOpenapiClient
```

Comment:
Hm what is this `include` actually doing for us?  This module isn't really doing anything besides defining a `.configure` method on `AutosdeOpenapiClient` which it doesn't look like you're using